client/views/monitorings/birds.html

```diff
@@ -3,12 +3,12 @@
       confirm-on-exit="smartform.$dirty"
       confirm-message-window="{{'BIRDS_DETAIL_UNSAVED_DATA_ALERT_TITLE' | translate}}"
       confirm-message-route="{{'BIRDS_DETAIL_UNSAVED_DATA_ALERT_MESSAGE' | translate}}">
+  <div class="actions actions-detail">
```

Comment:
actions-detail definition is missing in styles file

Reply:
It's not needed for now